knative/knative-serving-install/base/deployment.yaml

```diff
@@ -74,12 +77,31 @@ spec:
       serviceAccountName: controller
       terminationGracePeriodSeconds: 300
 ---
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```

Comment:
Wouldn't it be better to have this in the same kustomize definition file hpa.yaml?

Reply:
fixed.